server/commands/accountProvisioner.ts

```diff
@@ -21,18 +21,18 @@ type Props = {
     username?: string;
   };
   team: {
-    id?: string;
+    teamId?: string;
```

Comment:
Not really sure why this change – can we just add the JSDoc comment to the property if you're finding it ambigous.

Reply:
(I added the parameter in the PR right before this one, so it's more my changing my mind on naming conventions). We specify `authenticationProviderId` no matter how it's nested, so this is more consistent and unambiguous